src/content/docs/dropins/cart/cart-slots.mdx

```diff
@@ -1,13 +1,14 @@
 ---
```

Comment:
The [Overview page](https://github.com/commerce-docs/microsite-commerce-storefront/pull/125) can be updated to include the cart slots